_posts/2016-03-12-make-your-data-obey.markdown

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title: "Make Your Data Obey"
+date: 2016-03-12 10:00:00
+categories: obey data validation modelling
+author: kent.safranski
+cover: /assets/images/covers/obey.jpg
+description: Data is what programming is all about. Applications would be nothing without that beautiful i/o of data moving between modules, services, and data storage. But data is nothing if it's not consistant.
+comments: true
+disqus: true
+---
+
+Data is what programming is all about. Applications would be nothing without that beautiful i/o of data moving between modules, services, and data storage. But data is nothing if it's not consistant. One often ill-attended area of application development is data modelling and validation; most applications have a layer for addressing this, but a lot fall short of really implementing a solid solution.
```

Comment:
That last line is a little unsubstantiated-- I could see others calling you out for a strongman argument here.nwe could claim we believe data validation is a deceptively difficult task and that it can be done better, but I wouldn't make a blanket statement accusing others of doing it wrong without sources.